lib/BC/Util.cpp

```diff
@@ -1465,6 +1464,9 @@ llvm::GlobalVariable *DeclareVarInModule(llvm::GlobalVariable *var,
     return llvm::dyn_cast<llvm::GlobalVariable>(moved_var);
   }
 
+  // NOTE(alex): This looks difficult to reproduce without querying `PointerElementType`.
```

Comment:
This one seems a bit tricky. The variables seem to be pointers but we want to move the underlying variable that the pointers is pointing at to the other module. That information just doesn't seem to be there with opaque pointers.

Reply:
Seems like I can do `llvm::GlobalValue::getValueType` as a replacement.